src/bio/smd/smd_internal.h

```diff
@@ -54,7 +55,7 @@ struct smd_df {
 /* SMD store (DRAM structure) */
 struct smd_store {
 	struct umem_instance	ss_umm;
-	pthread_rwlock_t	ss_rwlock;
+	ABT_mutex		ss_mutex;
```

Comment:
Should we consider having two diff mutexes in case there is access to two trees simultaneously from ULTs? I know its not Perf path but just think we might lock when not explicitly required.

Reply:
Chatted with Niu about this, we will revisit this in future when SMD expands and this becomes a bottleneck